parlai/core/image_featurizers.py

```diff
@@ -0,0 +1,145 @@
+# All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+import parlai.core.build_data as build_data
+
+import torch
+from torch.autograd import Variable
+import torchvision
+import torchvision.transforms as transforms
+import torch.nn as nn
+
+import os
+import copy
+import numpy as np
+from PIL import Image
+
+_greyscale = '  .,:;crsA23hHG#98&@'
+
+class ImageLoader():
+	"""Extract image feature using pretrained CNN network.
+	"""	
+	@staticmethod
+	def add_cmdline_args(argparser):
+		argparser.add_arg('--image-cnntype', type=str, default='resnet152',
```

Comment:
actually: `--image-mode` can cover this, unless you can think of a case where image-mode might be set to something that would clash? otherwise let's delete this param